src/components/PageSection/index.tsx

```diff
@@ -70,7 +89,13 @@ const PageSection: React.FC<PageSectionProps> = ({
           dividerComponent={dividerComponent}
         />
       )}
-      <BackgroundColor background={background} index={index} hasZeroMargin={hasZeroMargin} {...props}>
+      <BackgroundColor
+        background={background}
+        index={index}
+        padding={getPadding()}
+        hasZeroMargin={hasZeroMargin}
```

Comment:
Just a thought. This prop is oddly specific. Would it be cleaner to pass in the margin and calculate whether it needs it in the parent component?

Reply:
Hmmn this was a useful pause for me to reflect on exactly why this negative top & bottom margins are needed. It's basically 'pulling' two page sections into one another, so that they overlap at the 'curved' SVG, giving the layered effect. The curved section's height changes as the page scales, hence why the margin has a few breakpoints- getting larger as the screen gets bigger. 

I think giving the parent control of the margin will massively clean up the components' margins, which are frankly.. mental. 

Let me make the change here and see what you think.